tools/cibadmin.c

```diff
@@ -94,7 +93,9 @@ print_xml_output(xmlNode * xml)
         }
 
     } else {
-        pcmk__xml2fd(STDOUT_FILENO, xml);
+        char *buffer = dump_xml_formatted(xml);
+        fprintf(stdout, "%s", pcmk__s(buffer, "<null>\n"));
```

Comment:
Since we know xml is not NULL, buffer can't be NULL here since the library functions would have asserted (similar for the other places)